### DISCUSSION TITLE
Error 500 server api onchain wallet transactions

### DISCUSSION THREAD

**Discussion by caalber999** [btcpayserver/btcpayserver#4900] [General]
When I consume this endpoint, it generates the 500 server error and I am carrying out the process that is.

https://mainnet.demo.btcpayserver.org/api/v1/stores/{storeId}/payment-methods/onchain/{cryptoCode}/wallet/transactions/{transactionId}

![Screenshot_7](https://user-images.githubusercontent.com/10522431/233659191-49b5e63e-95dc-4a98-9dc6-80d02a709cda.jpg)

**Comment by mp21-org** [comment]
Your transaction-id seems odly short. Could it be that you are trying with a Invoice ID instead of a BTC TX ID?
Still internal server error seems like an odd reaction to an invalid transaction id: So that part should probably be considered a bug.

Would be great if you could create an [issue](https://github.com/btcpayserver/btcpayserver/issues) for that.

**Reply by caalber999** [reply]
how can i get BTC TX ID?

**Reply by mp21-org** [reply]
Maybe you can use this call?
https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreOnChainWallets_ShowOnChainWalletTransactions
